### DISCUSSION TITLE
Generate site from many dsl files

### DISCUSSION THREAD

**Discussion by yadanilov19** [avisi-cloud/structurizr-site-generatr#449] [General]
Hi everyone!

I'm trying to generate a site from a model that has multiple files for each microservice and one at the root called "landscape.". 
It's pretty much similar to this example: https://github.com/structurizr/examples/tree/main/enterprise/src/main/resources/example1

Does anyone did that? I just can't figure out how to make it... Pls help

**Comment by dooch** [comment]
Is there an update on this question? I am trying to achieve the same result.

**Comment by jp7677** [comment]
This project doesn't merge several `dsl` files itself like the code that is behind the examples you linked. That said, using `includes`, see https://docs.structurizr.com/dsl/includes , should just work.

**Reply by dirkgroot** [reply]
In addition to this: `extends` (see https://docs.structurizr.com/dsl/language#workspace) should just work as well.

**Comment by thomasvdr** [comment]
Hi, 
On the same topic, I think it would be great to have support for multiple workspaces, not to merge them but to navigate between them.
For instance, we have several workspaces, one per business domain, containing all the softwareSystems of the corresponding business domain. 
What would be nice is to be able to navigate through the differents workspaces from the same site. Could be for instance from a menu or a dropdown in the Header of the site.
Currently I think we have to deploy a site for each workspace our organization have.
